test/libsolidity/SemanticTest.cpp

```diff
@@ -321,7 +355,7 @@ void SemanticTest::printSource(ostream& _stream, string const& _linePrefix, bool
 void SemanticTest::printUpdatedExpectations(ostream& _stream, string const&) const
 {
 	for (auto const& test: m_tests)
-		_stream << test.format("", true, false) << endl;
+		_stream << test.format("", !m_gasCostFailure, false);
```

Comment:
Being able to see the names of these boolean parameters would be very helpful when reviewing:
```suggestion
		_stream << test.format("", /* _renderResult = */ !m_gasCostFailure, /* _highlight = */ false);
```

Reply:
Added. I do not have this habit as my IDE (CLion) adds annotations for me

Reply:
Yeah, it's not that much of a problem when I'm using an editor and can jump to definition but I can't do that on github :)

And it's not something I need all the time but in this function in particular the parameters were a complete mystery to me until I checked :)

Reply:
Completely agree with you on this :)